### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by paolotron** [Kami-code/dexart-release#10]
Hi I tested your evaluation script but it seems that it is non-deterministic even if testing with the same exact checkpoint
Running the following command 3 times:
```python examples/evaluate_policy.py --task_name=laptop --checkpoint_path assets/rl_checkpoints/laptop.zip --eval_per_instance 10```
I obtain the following results
1. reward mean: 227 success rate: 0.57
2. reward mean: -0.12 success rate: 0.48
3. reward mean:  -0.053 success rate: 0.38
Is it normal that the evaluation changes across different evaluations? the policy is always the same.
Also the results for the laptop task don't match the results of the paper, am I doing something wrong?

**Comment by Kami-code** [comment]
Hi @paolotron . Thanks for using our benchmark. Yes, the evaluation is non-deterministic because we randomize position and orn in the evaluation scripts [here](https://github.com/Kami-code/dexart-release/blob/2923c6981e6fcb09ccd55f47d8986e33d9aa35af/examples/evaluate_policy.py#L42C1-L43C46). You can either change the logic to a deterministic one or increase the eval_per_instance param to get the success rate with lower variance. Also, since we still updated the metrics and simulation code after reporting the results in the paper. We can guarantee that the experiments in the paper are in the same training settings but we cannot guarantee the comparability of your experimental results and the values in the paper. Also, the values in the paper are averaged between three seeds of evaluations, so I recommend you train your policy and the baseline under the same configuation (also using the same random seeds) to get a fair comparision.

**Comment by Kami-code** [comment]
Hi @paolotron. To ensure the reproducibility of our benchmark. We are planning to release all the checkpoints for customized evaluation. Maybe in a few weeks, too much random stuff to organize. :)

**Comment by Kami-code** [comment]
Hi @paolotron . I update the [evaluate.py](https://github.com/Kami-code/dexart-release/blob/main/examples/evaluate_policy.py) to a deterministic one and reported the results [here](https://github.com/Kami-code/dexart-release#main-results) with an increased eval_per_instance of 100 per instance per seed.  I think the results will remain the same on different machines. You are welcome to cross-check it on your own machine.

**Comment by Kami-code** [comment]
Hi @paolotron . We have updated all the checkpoints in assets.zip. You can check the results in [Main Results](https://github.com/Kami-code/dexart-release#main-results).
